Quaternion/Quaternion.py

```diff
@@ -100,10 +99,24 @@ class Quat(object):
       * a 4 element array (expects x,y,z,w quat form)
       * a 3 element array (expects ra,dec,roll in degrees)
       * a 3x3 transform/rotation matrix
+    N x those types :
+     * an N x 4 element array ( N by x,y,z,w quat form)
+     * an N x 3 element array ( N by ra, dec, roll in degrees,
+       if N == 3, the optional 'intype = 'equatorial' may be used to differentiate,
+       this from a transform matrix)
+     * an N x 3x3
+
+   :param intype: optional type to describe input attitude
+
+   ``intype`` may be:
+     * transform
+     * equatorial
+     * quaternion
+
 
     """
 
-    def __init__(self, attitude):
+    def __init__(self, attitude, intype=None):
```

Comment:
The idea of "intype" is really to let us easily keep the default behavior that a 3x3 is treated as a transform and not a weird three-row ra,dec,roll initialization.  We could probably also check for a transform matrix vs ra,dec,roll entries by value, but the initialization option seemed a safe convention at the time.